samples/bluetooth/hci_uart/src/main.c

```diff
@@ -70,22 +71,39 @@ static int h4_read(const struct device *uart, uint8_t *buf, size_t len)
 
 static bool valid_type(uint8_t type)
 {
-	return (type == H4_CMD) | (type == H4_ACL);
+	return (type == H4_CMD) | (type == H4_ACL) | (type == H4_ISO);
 }
 
-/* Function assumes that type is validated and only CMD or ACL will be used. */
+/* Function expects that type is validated and only CMD, ISO or ACL will be used. */
```

Comment:
Nitpick.com: Would be nice if CMD, ACL and ISO are referenced in the same order on line 74, 77 and the switch on 80 and 97

Reply:
Not sure if you want any specific order, but it should be ordered as CMD, ISO, ACL everywhere now :)